modules/core/src/main/java/org/apache/fluo/recipes/core/map/CollisionFreeMap.java

```diff
@@ -194,12 +199,11 @@ void process(TransactionBase tx, Bytes ntfyRow, Column col) throws Exception {
       tx.setWeakNotification(ntfyRow, col);
     }
 
-    byte[] dataPrefix = ntfyRow.toArray();
```

Comment:
It should.   I added a sanity check at the beginning of process to ensure it starts with updatePrefix